src/gc/env/gcenv.os.h

```diff
@@ -7,6 +7,17 @@
 #ifndef __GCENV_OS_H__
 #define __GCENV_OS_H__
 
+#ifdef Sleep
+// This is a funny workaround for the fact that "common.h" defines Sleep to be
```

Comment:
@jkotas  Do you want me to create an issue?

My tip has these #define for Sleep...
```
src/gc/gc.h:// This is a funny workaround for the fact that "common.h" defines Sleep to be
src/gc/gc.h:// However, GCToOSInterface defines a function called Sleep, which (due to this define) becomes
src/inc/clrhost.h:#define Sleep(dwMilliseconds) \
src/vm/comsynchronizable.h:#define Sleep(a) Dont_Use_Sleep(a)
src/vm/ecalllist.h:#define Sleep(a) Dont_Use_Sleep(a)
```